examples/trading/api/prisma/migrations/migration_lock.toml

```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
```

Comment:
```suggestion
# Do not edit this file manually
```
Is this auto-generated? I'd really like to remove this word here if it is.

Reply:
Yeah it's auto-generated and will probably be written over even if we do any change.